CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Look at all the fail tests. Example of platform without latest CMake 3.0...

Depending on the latest isn't always best if you want as much portability as possible.

Solve this by doing `cmake_policy(SET CMP0042 OLD)` instead